public/mix-manifest.json

```diff
@@ -1,7 +1,10 @@
 {
     "/js/app.js": "/js/app.js",
     "/css/app.css": "/css/app.css",
+    "/images/download-stats-black.svg": "/images/download-stats-black.svg",
+    "/images/download-stats-white.svg": "/images/download-stats-white.svg",
     "/images/download-stats.svg": "/images/download-stats.svg",
```

Comment:
question: As you renamed one usage of the `download-stats.svg` file to `download-stats-white.svg`, is this line here still needed?